popde/density_estimate.py

```diff
@@ -313,7 +326,7 @@ def plot_2d_contour(self, dim1, dim2, slice_dims=None, slice_values=None, num_po
 
         # Create the contour plot
         zz = z.reshape(xx.shape)
-        fig = utils_plot.simple2Dplot(xx, yy, zz, xlabel=dim1, ylabel=dim2, title=f'KDE sliced along {slice_dims} at {slice_values})')
+        fig = utils_plot.simple2Dplot(xx, yy, zz, xlabel=dim1, ylabel=dim2, title=f'KDE sliced along {slice_dims} at {slice_values})', show_plot=True)
```

Comment:
why has this been changed?

Reply:
I removed it. I was testing the code for examples and mistakenly commit this. Fixed it now